### PR TITLE
blog: interacting with Google sheet from Node-RED

### PR DESCRIPTION
**Pull request by sumitshinde-84**
## Checklist

 - [ ] I have read the [contribution guidelines](https://github.com/FlowFuse/flowfuse/blob/main/CONTRIBUTING.md)
 - [ ] I have considered the performance impact of these changes
 - [ ] Suitable unit/system level tests have been added and they pass
 - [ ] Documentation has been updated

### CODE REVIEW

File: src/blog/2024/06/interacting-with-google-sheet-from-node-red.md [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+3. Enter the environment variable added for the private key in the "creds" field and click "Add."
+4. Go to the Google Sheet you want to interact with and copy its ID from the URL. The URL will be in this format: `https://docs.google.com/spreadsheets/d/<id_of_sheet>/`
+5. Return to your Node-RED instance, double-click on the GSheet node again, and paste the spreadsheet ID into the "SpreadsheetID" field.
+6. Enter the range of cells you want to work with using the syntax `<sheetname!first-cell-name:last-cell-name>`. For example, use `Sheet1!A1:C3` to specify that you are working with the "Sheet1" tab, starting from cell "A1" to cell "C3". This syntax allows you to define specific ranges such as a row (`A1:A5`), a column (`A1:E1`), or a block (`A1:C3`) within the spreadsheet.
+
+### Writing Data to Cells
+
+For demonstration purposes, I will write simulated sensor data which includes a timestamp and sensor data.
+
+1. Drag the Inject node onto the canvas, and set `msg.payload` to `[$moment().format(), $random() * 100]` as a JSONata expression, and set it to repeat every 3 seconds of interval.
```
Comment (ZJvandeWeg) [inline src/blog/2024/06/interacting-with-google-sheet-from-node-red.md:106]: Can we next time just inject and change node? For now this is fine, but not the `low-code` way.

File: src/blog/2024/06/interacting-with-google-sheet-from-node-red.md [hunk 2]
```diff
@@ -0,0 +1,150 @@
+---
+title: Interacting with Google Sheet from Node-RED
```
Comment (ZJvandeWeg) [inline src/blog/2024/06/interacting-with-google-sheet-from-node-red.md:2]:
```suggestion
title: Interacting with Google Sheets from Node-RED
```

File: src/blog/2024/06/interacting-with-google-sheet-from-node-red.md [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+subtitle: Guide to learn how to write, read, update and delete data in Google sheet using Node-RED.
+description: Learn how to interact with Google Sheet from Node-RED to write, read , update and delete data.
```
Comment (ZJvandeWeg) [inline src/blog/2024/06/interacting-with-google-sheet-from-node-red.md:4]:
```suggestion
description: Learn how to interact with Google Sheet from Node-RED to write, read, update and delete data.
```

File: src/blog/2024/06/interacting-with-google-sheet-from-node-red.md [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+
+Before proceeding, make sure you have installed the following node from the pallet manager.
+
+- [node-red-contrib-google-sheets](https://flows.nodered.org/node/node-red-contrib-google-sheets)
+
+## Interacting with Google Sheets with Node-RED
+
+When writing this blog, I watched some videos on the internet about how to integrate Google Sheets with our application with traditional programming. I noticed that almost all of the videos demonstrate complex steps and involve a lot of coding. Fortunately, as a Node-RED developer, you won't have to go through such lengthy processes.
+
+### Creating a Google service account
```
Comment (ZJvandeWeg) [inline src/blog/2024/06/interacting-with-google-sheet-from-node-red.md:34]: Not sure what these lines are for? :) It's a guide, lets focus on getting things done

```suggestion
```

Comment (sumitshinde-84) [reply src/blog/2024/06/interacting-with-google-sheet-from-node-red.md:34]: should we also remove the section's title ?

**Review by ZJvandeWeg — APPROVED** [review]

File: src/blog/2024/06/interacting-with-google-sheet-from-node-red.md (hunk 4 above)
Comment (ZJvandeWeg) [reply src/blog/2024/06/interacting-with-google-sheet-from-node-red.md:34]: I think so, but up to you.

**Comment by ZJvandeWeg** [comment]
Approved if we have a tile. @Yndira-E we should probably pick a tool like canva to create tiles so Sumit and others can also create them

**Pull request merged**